lib/puppet/pops/issues.rb

```diff
@@ -58,7 +58,8 @@ def format(hash ={})
   # in the issue's message producing block.
   # @api private
   #
-  class MessageData
+  class MessageData < Puppet::Pops::LabelProvider
```

Comment:
After discussion we decided to change the label method instead so it takes *args. That fixes the actual problem and leaves the rest alone.